src/_interfaces/reporter.js

```diff
@@ -24,7 +24,7 @@ import Span from '../span.js';
 declare interface Reporter {
     report(span: Span): void;
     close(callback: ?Function): void;
-    setProcess(serviceName: string, tags: Array<Tag>): void;
+    name(): string;
```

Comment:
Only remote, and composite reporter really need setProcess though, because only they delegate anything to the sender.